accesscontrol_usermanagement/configuring-sso-for-argo-cd-using-keycloak.adoc

```diff
@@ -11,7 +11,7 @@ After the {gitops-title} Operator is installed, Argo CD automatically creates a
 [id="prerequisites_configuring-sso-for-argo-cd-using-keycloak"]
 == Prerequisites
 * Red Hat SSO is installed on the cluster.
-* {gitops-title} Operator is installed on the cluster.
+* You have installed the {gitops-title} Operator on your {OCP} cluster.
```

Comment:
This phrasing makes the middle item no longer parallel construction to the others. Does this work?
```suggestion
* The {gitops-title} Operator is installed on your {OCP} cluster.
```

Reply:
Fixed.